### PR TITLE
Add sectionheader aa1 to master

### LINKED ISSUE #4: There should be a SectionHeader
**Is your feature request related to a problem? Please describe.**
It would be nice to implement the wish of the client. Which is a gradient section with his logo in the middle of it.

**Describe the solution you'd like**
Add SectionHeader to the current environment.

### PR DESCRIPTION
**Pull request by petschna**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no SectionHeader to introduce one of the wishes of the client. #4 

**What is the new behavior (if this is a feature change)?**
- Now there is a SectionHeader which implements the wish of the client. Closes #4

### CODE REVIEW

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

File: src/components/organisms/SectionHeader/header.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  .imageBox .hoverImg {
+    position: absolute;
+    left: 0;
+    top: 0;
+    display: none;
+  }
+  
+  .imageBox:hover .hoverImg {
+    display: block;
+  }
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/header.scss]:
```suggestion
  }

```

File: src/components/organisms/SectionHeader/header.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#header .view {
+    min-height: 70vh !important;
+    height: unset !important;
+  }
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/header.scss]:
```suggestion
}
```

File: src/components/organisms/SectionHeader/header.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    margin-left: 1.3em;
+}
+
+.imageBox {
+    position: relative;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+  }
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/header.scss]:
```suggestion
}
```

File: src/components/organisms/SectionHeader/header.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  
+  .imageBox .hoverImg {
+    position: absolute;
+    left: 0;
+    top: 0;
+    display: none;
+  }
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/header.scss]:
```suggestion
}
```

File: src/components/organisms/SectionHeader/header.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+.imageBox {
+    position: relative;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+  }
+  
+  .imageBox .hoverImg {
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/header.scss]:
```suggestion
.imageBox .hoverImg {
```

File: src/components/organisms/SectionHeader/index.jsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                  <div class="hoverImg">
+                    <img
+                      className="img-fluid"
+                      src={logowhite}
+                      alt=""
+                      width="500px"
+                    />
+                  </div>
+                </div>
+
```
Comment (kleberbaum) [inline src/components/organisms/SectionHeader/index.jsx]:
```suggestion
```

**Review by kleberbaum — CHANGES_REQUESTED** [review]
Should be ok to merge after a few changes.

**Review by petschna — COMMENTED** [review]
Should be ok to merge.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**